src/SFML/CMakeLists.txt

```diff
@@ -31,9 +31,6 @@ elseif(SFML_OS_ANDROID)
     list(APPEND CMAKE_LIBRARY_PATH "${PROJECT_SOURCE_DIR}/extlibs/libs-android/${CMAKE_ANDROID_ARCH_ABI}")
 endif()
 
-# define the path of our additional CMake modules
-set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake/Modules/")
```

Comment:
How does CMake find the scripts in: https://github.com/SFML/SFML/tree/master/cmake/Modules ?

Or can we get rid of those?

Reply:
That same code also exists [here](https://github.com/SFML/SFML/blob/master/cmake/Macros.cmake#L406).

I suppose it would make more sense to keep this line and remove the one deep in Macros.cmake instead so I'll amend this PR to do that instead. That will keep the code more readable.